Assets/MixedRealityToolkit-Examples/Demos/Boundary/Scripts/BoundaryVisualizationDemo.cs

```diff
@@ -5,7 +5,6 @@
 using Microsoft.MixedReality.Toolkit.Core.EventDatum.Boundary;
 using Microsoft.MixedReality.Toolkit.Core.Interfaces.BoundarySystem;
 using Microsoft.MixedReality.Toolkit.Core.Services;
-using Microsoft.MixedReality.Toolkit.Core.Utilities;
```

Comment:
while removing this line feels like a good cleanliness change, it doesn't feel like part of this change...

(plus other similar changes)

Reply:
I heavily adjusted the namespaces and resharper cleaned them up automatically. I don't think it's worth the effort to pull them out separately.